packages/lambda-xyz/src/__test__/wmts-capability.test.ts

```diff
@@ -0,0 +1,73 @@
+import * as o from 'ospec';
+import { buildWmtsCapability } from '../wmts-capability';
+
+const findElm = (ans: string[], text: string): number[] => {
+    const sidx = ans.findIndex(i => i === '<' + text + '>');
+    const eidx = ans.findIndex(i => i === '</' + text + '>');
+    return [sidx, eidx, eidx - sidx];
+};
+
+o.spec('wmts', () => {
+    o('should build capabiltiy xml for tileset and projection', () => {
```

Comment:
instead of string manipulations you could expose the function to give you the VDOM and run tests on that aswell as it might be easier.

Reply:
I did this but testing correct output was easier as a string

Reply:
OK, but IMHO any production release should be a parameterised template.